package.json

```diff
@@ -25,7 +25,6 @@
     "env:prod": "node ./env-mgt/set-env.js prod",
     "dev": "vite",
     "build:ui": "vite build",
-    "build:css:watch": "npx tailwindcss -i ./src/client/assets/input.css -o ./src/client/assets/output.css --watch",
```

Comment:
Vite does not compile Tailwind, I don't see why you believe Vite executes Tailwind. What am I missing here?

Reply:
The CSS processing happens automatically as part of your normal development and build processes.

Reply:
How exactly? `npx tailwind` watches for changes in the files, identifies the classes you are using, takes the settings from the `input` file and compiles the `output` file. If it's not running, how is the CSS processing happening?